conf/.env

```diff
@@ -0,0 +1,47 @@
+#
+# You should NOT use this file in production, but instead move the environment variables to your webserver configuration.
+# The .env file is only existing to simplify the initial setup!
+#
+
+#================================================================================
+# Configure your database connection and set the correct server version.
+#
+# You have to replace the following values with your defaults:
+# - the version "5.7" (the database server version, examples for MySQL and MariaDB below)
+# - the database username "user"
+# - the password "password" for "user"
+# - the database schema "database"
+# - you might have to adapt port "3306" and server IP "127.0.0.1" as well
+#
+# For MySQL that would be "serverVersion=5.7" as in:
+#    DATABASE_URL=mysql://user:password@127.0.0.1:3306/database?charset=utf8mb4&serverVersion=5.7
+#
+# For MariaDB it would be "serverVersion=mariadb-10.5.8":
+#    DATABASE_URL=mysql://user:password@127.0.0.1:3306/database?charset=utf8mb4&serverVersion=mariadb-10.5.8
+#
+DATABASE_URL=mysql://__DB_USER__:__DB_PWD__@127.0.0.1:3306/__DB_NAME__?charset=utf8mb4&serverVersion=mariadb-10.5.19
+
+#================================================================================
+# The full documentation can be found at https://www.kimai.org/documentation/emails.html
+#
+# Email will be sent with this address as sender:
+MAILER_FROM=admin@__DOMAIN__
+# Email connection (disabled by default) - see documentation for the format
+MAILER_URL=smtp://localhost:25?encryption=&auth_mode=
```

Comment:
Note for the future :  we may want mails to be dkim-signed and may use `MAILER_URL="smtp://<user>:<password>@localhost:25?encryption=&auth_mode="` syntax (cf their doc) but that requires packaging v2 with `allow_email=true` on the system user resource